reactive-streams/src/main/scala/fs2/interop/reactivestreams/StreamSubscription.scala

```diff
@@ -41,7 +41,8 @@ private[reactivestreams] final class StreamSubscription[F[_], A](
     cancelled: SignallingRef[F, Boolean],
     sub: Subscriber[A],
     stream: Stream[F, A],
-    dispatcher: Dispatcher[F]
+    startDispatcher: Dispatcher[F],
+    requestDispatcher: Dispatcher[F]
```

Comment:
We need two `Dispatcher`s for `StreamSubscription`, but they can both be sequential.

The first is used in `unsafeStart()` and actually runs the subscription. So it's blocked on that.

The second is used for handling requests and canceling.